man/pt.man

```diff
@@ -0,0 +1,61 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - relatório de software não grátis instalado
+.SH SYNOPSIS
+\fBvrms-rpm\fR \fI[OPTION]\fR ...
+.SH DESCRIPTION
+Este programa é um clone do programa original
+\ fBvrms \ fR ("\ fBvirtual Richard M. Stallman \ fR")
```

Comment:
`\fB` and `\fR` are formatting marks - they switch between bold and regular text. As such, they must be kept as-is, without adding a space after the backslash.